CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.4.0]
```

Comment:
```suggestion
## [0.4.0] includes BREAKING CHANGES
```

Reply:
following @mariux scheme here :)

Reply:
@mariux @soerenmartius @zied-elouaer we should agree on the format as last time we also brought `Internal change` from a changelog and now it seems a little hectic. `BREAKING CHANGE` in the content or next to the version? having it numerous places seems redundant or noisy

Reply:
makes sense, let's have @mariux commenting on this :)